editor/src/clj/editor/code/view.clj

```diff
@@ -2356,116 +2365,40 @@
                    (data/split-selection-into-lines (get-property view-node :lines)
                                                     (get-property view-node :cursor-ranges))))
 
-(handler/defhandler :select-up :code-view
-  (run [view-node] (move! view-node :selection :up)))
-
-(handler/defhandler :select-down :code-view
-  (run [view-node] (move! view-node :selection :down)))
-
-(handler/defhandler :select-left :code-view
-  (run [view-node] (move! view-node :selection :left)))
-
-(handler/defhandler :select-right :code-view
-  (run [view-node] (move! view-node :selection :right)))
-
-(handler/defhandler :prev-word :code-view
-  (run [view-node] (move! view-node :navigation :prev-word)))
-
-(handler/defhandler :select-prev-word :code-view
-  (run [view-node] (move! view-node :selection :prev-word)))
-
-(handler/defhandler :next-word :code-view
-  (run [view-node] (move! view-node :navigation :next-word)))
-
-(handler/defhandler :select-next-word :code-view
-  (run [view-node] (move! view-node :selection :next-word)))
-
-(handler/defhandler :beginning-of-line :code-view
-  (run [view-node] (move! view-node :navigation :line-start)))
-
-(handler/defhandler :select-beginning-of-line :code-view
-  (run [view-node] (move! view-node :selection :line-start)))
-
-(handler/defhandler :beginning-of-line-text :code-view
-  (run [view-node] (move! view-node :navigation :home)))
-
-(handler/defhandler :select-beginning-of-line-text :code-view
-  (run [view-node] (move! view-node :selection :home)))
-
-(handler/defhandler :end-of-line :code-view
-  (run [view-node] (move! view-node :navigation :end)))
-
-(handler/defhandler :select-end-of-line :code-view
-  (run [view-node] (move! view-node :selection :end)))
-
-(handler/defhandler :page-up :code-view
-  (run [view-node] (page-up! view-node :navigation)))
-
-(handler/defhandler :select-page-up :code-view
-  (run [view-node] (page-up! view-node :selection)))
-
-(handler/defhandler :page-down :code-view
-  (run [view-node] (page-down! view-node :navigation)))
-
-(handler/defhandler :select-page-down :code-view
-  (run [view-node] (page-down! view-node :selection)))
-
-(handler/defhandler :beginning-of-file :code-view
-  (run [view-node] (move! view-node :navigation :file-start)))
-
-(handler/defhandler :select-beginning-of-file :code-view
-  (run [view-node] (move! view-node :selection :file-start)))
-
-(handler/defhandler :end-of-file :code-view
-  (run [view-node] (move! view-node :navigation :file-end)))
-
-(handler/defhandler :select-end-of-file :code-view
-  (run [view-node] (move! view-node :selection :file-end)))
-
```

Comment:
UI handlers for "read-only" actions, shared with the Console view, have been moved to `fundamental-read-only-handlers` later in the file.